src/applications/virtual-agent/tests/Chatbox.unit.spec.js

```diff
@@ -923,6 +923,74 @@ describe('App', () => {
           expect(window.ReactDOM).to.eql(ReactDOM);
         });
       });
+      describe('Timers', () => {
+        const locationReload = window.location;
+
+        afterEach(() => {
+          window.location = locationReload;
+        });
+        it('Will reload the page after 60 minutes of first render', () => {
+          sandbox.useFakeTimers({
+            now: 0,
+            toFake: ['setTimeout'],
+          });
+          const unacknowledgedUserStore = {
+            initialState: {
+              featureToggles: { loading: false },
+              virtualAgentData: { termsAccepted: false },
+              user: {
+                login: { currentlyLoggedIn: true },
+                profile: { userFullName: { first: 'Steve' } },
+              },
+            },
+            reducers: virtualAgentReducer,
+          };
+          renderInReduxProvider(
+            <Chatbox {...defaultProps} />,
+            unacknowledgedUserStore,
+          );
+          window.location = { reload: sinon.stub() };
+
+          sandbox.clock.tick(60 * 60 * 1000 - 1);
+          expect(window.location.reload.called).to.be.false;
+          sandbox.clock.tick(1);
+          expect(window.location.reload.called).to.be.true;
+        });
+      });
+    });
+  });
+  describe('when receiving a new message event', () => {
+    it('resets timer', async () => {
+      const fakeClearTimeout = sandbox.stub(global, 'clearTimeout');
+      sandbox.useFakeTimers({ now: 0, toFake: ['setTimeout'] });
+      const unacknowledgedUserStore = {
+        initialState: {
+          featureToggles: { loading: false },
+          virtualAgentData: { termsAccepted: false },
+          user: {
+            login: { currentlyLoggedIn: true },
+            profile: { userFullName: { first: 'Steve' } },
+          },
+        },
+        reducers: virtualAgentReducer,
+      };
+      renderInReduxProvider(
+        <Chatbox {...defaultProps} />,
+        unacknowledgedUserStore,
+      );
+      const incomingActivityEvent1 = new Event('bot-incoming-activity');
+      const testNumber = 7;
+      incomingActivityEvent1.data = testNumber;
+      window.dispatchEvent(incomingActivityEvent1);
+      sandbox.clock.tick(1);
+      expect(fakeClearTimeout.called).to.be.false;
```

Comment:
Curious about this second test case. Want to understand why the significance of the window event `data` property and why we expect the timer to not be called at first.